apps/preview/nuxt/pages/showcases/Banners/Hero.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div
+    :style="variables"
+    class="md:flex md:flex-row-reverse min-h-[600px] max-w-[1536px] mx-auto bg-[url:var(--sf-img-mobile)] md:bg-[url:var(--sf-img-desktop)] bg-center bg-cover bg-no-repeat"
+  >
+    <div class="flex flex-col md:basis-2/4 md:items-stretch md:overflow-hidden">
+      <img :src="headphones" alt="Headphones" class="h-full object-cover object-left" />
+    </div>
+    <div class="p-4 md:p-10 md:flex md:flex-col md:justify-center md:items-start md:basis-2/4">
+      <p class="typography-text-xs md:typography-text-sm font-bold tracking-widest text-neutral-500">Feel the music</p>
+      <h2 class="typography-headline-2 md:typography-headline-1 md:leading-[67.5px] font-bold mt-2 mb-4">
```

Comment:
Hero should be the very first section of the page. I think it should contain `<h1>` somewhere. We could make this heading `<h1>`. Also we could make the wrapper a `<header>` for the same reason.

Reply:
I think it depends and user will decide. Of course I can change it but I also took under consideration that there is already h1 tag on our docs site so didn't want to make a bad practise to have more than one of this tag. Do you think it won't influence and should be changed?

Reply:
All of our examples are displayed in an iframe, so they should not affect the SEO.